src/Actions/index.js

```diff
@@ -0,0 +1,12 @@
+export function getEvents() {
```

Comment:
This seems to be where the magic comes in to pull in the data from our backend

Reply:
You are right!